packages/neon-api/src/provider/neoscan/class.ts

```diff
@@ -1,16 +1,24 @@
 import { logging, rpc, settings, u, wallet } from "@cityofzion/neon-core";
-import { PastTransaction, Provider } from "../common";
+import { IAddressAbstract, PastTransaction, Provider } from "../common";
 import {
   getBalance,
   getClaims,
   getHeight,
   getMaxClaimAmount,
   getRPCEndpoint,
   getTransactionHistory,
+  getAddressAbstracts,
 } from "./core";
 const log = logging.default("api");
 
-export class Neoscan implements Provider {
+interface ProviderNeoScan extends Provider {
+  getAddressAbstracts: (
+    address: string,
+    page: number
+  ) => Promise<IAddressAbstract>;
+}
+
+export class Neoscan implements ProviderNeoScan {

```

Comment:
Why `ProviderNeoScan` but previously `DoraProvider`? Please stick to a convention. I would prefer `NeoscanProvider`.

Reply:
make a sense

Reply:
resolved in https://github.com/CityOfZion/neon-js/pull/751/commits/a93d3049df5306ee83254f402de0601bf2ea4107